src/IncidentReportView.js

```diff
@@ -0,0 +1,175 @@
+import React, {Component} from "react";
+import Select from "react-select";
+import {Container, Row, Col} from 'reactstrap';
```

Comment:
You can import this from react-bootstrap which is already used elsewhere as well

https://react-bootstrap.github.io/layout/grid/